boards/arc/nsim/Kconfig.board

```diff
@@ -0,0 +1,13 @@
+# Kconfig - DesignWare ARC nSIM simulated platform configuration
+#
+# Copyright (c) 2016, 2019 Synopsys, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+config BOARD_NSIM
+	bool "ARC nSIM simulator"
+	depends on SOC_NSIM
+	help
+	  The DesignWare ARC nSIM board is a virtual board based on
```

Comment:
How about:
```
          The DesignWare ARC nSIM board is a virtual board based on
	  the ARC nSIM simulator.  It demonstrates the ARC core features
	  and a console based on the hostlink feature of the nSIM simulator.
```

Reply:
Well nSIM simulates among other things legacy ARC UART which is used for serial console in Zephyr/Linux/U-Boot and this is a completely different beast as compared to so-called hostlink (which is just a mailbox in target's memory).

Makes sense?

Reply:
I was trying to clarify the English a bit.  You wrote, "It just has a console based on hostlink feature of nSIM simulator."  Is what I suggested incorrect?  Can you use the simulated UART for console output?

Reply:
@dbkinder Oops, sorry. I blindly fixed spelling of `nSIM` in original description (https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arc/nsim_em/Kconfig.board) and failed to realize description by its nature was misleading. Indeed we use simulated UART (ARC UART, see https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/serial/Kconfig.nsim).

So based on your suggestion I'll reword the comment like that:
> The DesignWare ARC nSIM board is a virtual board based on
> the ARC nSIM simulator.  It demonstrates the ARC core features
> and a console based on the legacy ARC UART model.